src/lib.rs

```diff
@@ -165,9 +165,9 @@
 #![doc(html_root_url = "https://docs.rs/log4rs/0.8")]
 #![warn(missing_docs)]
 
-extern crate arc_swap;
 #[cfg(feature = "antidote")]
 extern crate antidote;
+extern crate arc_swap;
```

Comment:
are we migrating to 2018 or remaining in 2015 edition? I know this syntax is valid in both.

Reply:
I would like to see us migrate to 2018 personally.